tests/terraform/runner/resources/example/example.tf

```diff
@@ -475,6 +475,28 @@ resource "aws_s3_bucket" "bridgecrew_cws_bucket" {
   }
 }
 
+resource "aws_s3_bucket" "dynamic ssee block as string" {
```

Comment:
@arielkru This is not sse block as string, it is a full dict. Does this pass or fail?
Please add another case to show the other case (fail / pass)